src/lms-ds-loader/edfi_lms_ds_loader/scripts/mssql/initialize_lms_database.sql

```diff
@@ -0,0 +1,13 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF NOT EXISTS (SELECT * FROM sys.schemas WHERE name = N'lms')
+EXEC sys.sp_executesql N'CREATE SCHEMA lms';
+

```

Comment:
Note: if you try to run this script directly in SQL Management Studio then it might error out, because both statements are treated as a single batch. When the second statement runs, it will think that the `lms` schema doesn't exist yet. The normal solution is to add the word `GO` between statements. However, this won't work with the way that I'm running these files through SQL Alchemy: the files are split using `sqlparse`, which doesn't know about `GO`. Thus for this project the semi-colon separator is *essential*. I should mention this in the readme before finishing this ticket.

Reply:
done